src/components/Book.js

```diff
@@ -16,14 +16,21 @@ const Book = ({
   return (
     <>
       <div className="oneBook">
```

Comment:
- [ ] Your book is missing the demarcating line as provided in the template. Kindly add it.

Your Implementation | Zeplin Design
:---:|:---:
![benjp-demarc](https://user-images.githubusercontent.com/36542986/150310719-b9678516-1d46-40e8-8e31-dc332022b728.jpg) | ![benjp-orig](https://user-images.githubusercontent.com/36542986/150310641-27045207-d797-44e6-97dd-6586e9483bf5.jpg)